lib/assessment.rb

```diff
@@ -5,36 +5,52 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.max {|a, b| a.length <=> b.length}
+    end
   end
 
+  puts longest
+
   # Question 2
   # Write ruby code to return the sum of entries in the
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce{|a, b| a + b}
+    end
   end
 
+  puts sum
+
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    numbers.any? {|number| number%2 == 1}
```

Comment:
That would actually allow you to write 
`numbers.any?(&:odd?)` (or just `numbers.any?(:odd?)` in Ruby 2.0 and above)